README.md

```diff
@@ -6,12 +6,37 @@
 ## [`Dockerfile`](./Dockerfile)
 
 ## Usage with [docker](https://hub.docker.com/r/vierbergenlnars/bareos_exporter)
-1. Create a file containing your mysql password and mount it inside `/bareos_exporter/pw/auth`
-2. **(optional)** [Overwrite](https://docs.docker.com/engine/reference/run/#env-environment-variables) default args using ENV variables
-3. Run docker image as follows
+
+1. Create a .env file containing at least your database password using `DB_PASSWORD` variable.
+2. Run docker image as follows
+
+Using default variables (expected DB_PASSWORD)
```

Comment:
Default variables or custom variables are the same, except for some variable names set.

Can you drop this section and only keep the custom variables?